darwin/importer/formats/labelbox_schemas.py

```diff
@@ -0,0 +1,78 @@
+bounding_box = {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://darwin.v7labs.com/schemas/bounding_box",
+    "description": "Schema of a Bounding Box",
+    "title": "Bounding Box",
+    "default": {"top": 1.2, "left": 2.5, "height": 10, "width": 20},
+    "examples": [{"top": 0, "left": 0, "height": 10, "width": 20}],
+    "type": "object",
+    "properties": {
+        "top": {"type": "number"},
+        "left": {"type": "number"},
+        "height": {"type": "number"},
```

Comment:
Could we add some constraints, like e. g.: `"minimum": 0`?

Reply:
I don;t know if we have any constraints for `top` and `left`. I think the `minimum` one is a good start for `height` and `width` though!

Reply:
We most probably don't have constraints for top and left but yes, width and height can't really be negative.

Reply:
Fixed !